pom.xml

```diff
@@ -102,7 +102,7 @@
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-compat</artifactId>
```

Comment:
COuld you also move this dependency below, to group all the test dependencies together?